Cargo.toml

```diff
@@ -19,10 +18,14 @@ exclude = ["/ci/*", "/.travis.yml", "/appveyor.yml"]
 travis-ci = { repository = "rust-lang/libc" }
 appveyor = { repository = "rust-lang/libc", project_name = "rust-lang-libs/libc" }
 
+[dependencies]
+rustc-std-workspace-core = { version = "1.0.0", optional = true }
+
 [features]
 default = ["use_std"]
 use_std = []
 align = []
+rustc-dep-of-std = ['align', 'rustc-std-workspace-core']
```

Comment:
I don't know if this should enable `use_std` as well to be sure - it should be enabled by default anyways as long as libstd doesn't use `default-features=false`.